packages/viewer/src/lib/interactions/baseInteractionHandler.ts

```diff
@@ -201,8 +201,19 @@ export abstract class BaseInteractionHandler implements InteractionHandler {
   }
 
   protected beginDrag(event: BaseEvent): void {
+    const isMac = /(Mac|iPhone|iPod|iPad)/i.test(navigator.platform);
```

Comment:
looking at what we're doing in man-ui, it doesn't appear that we specifically distinguish based on the user agent. would be interested to pursue if there are ways we can continue, but also reviewing that code more to understand it better.

Reply:
looking at https://github.com/Vertexvis/manufacturing-ui/blob/master/src/js/modules/viewer/views/MonoscopicViewer.tsx#L779

which is using https://github.com/Vertexvis/manufacturing-ui/blob/master/src/js/modules/utils/platform.ts#L82

Reply:
We do for the pan tool (the same operation we are doing here). I got this code from https://github.com/Vertexvis/manufacturing-ui/blob/master/src/js/modules/utils/platform.ts#L121 and https://github.com/Vertexvis/manufacturing-ui/blob/master/src/js/modules/viewer_tools/views/common/PanModelTool.tsx#L14.

Reply:
I _think_ those are only used to decide whether to display the unicode icon in the tooltip or not, but isn't used when deciding which interaction mode to use.

Reply:
I looked into it some more and in legacy Connect, we actually pan when using either the command or control key on macs. https://github.com/Vertexvis/manufacturing-ui/blob/master/src/js/modules/viewer/views/MonoscopicViewer.tsx#L794 and https://github.com/Vertexvis/manufacturing-ui/blob/master/src/js/modules/utils/platform.ts#L28-L29. Would we rather do that?

Reply:
I would be ok with maintaining current behavior and using either of those keys for pan.

Reply:
Sounds good. I updated the code accordingly.